amclient/utils.py

```diff
@@ -94,7 +111,12 @@ def _call_url_json(url, params=None, method=METHOD_GET, headers=None, assume_jso
             err.response.status_code,
             err.response.reason,
         )
-        return errors.ERR_INVALID_RESPONSE
+        message = err.response.json if assume_json else err.response.text
```

Comment:
FYI before merging I'll be changing this line as follows:

```suggestion
        message = err.response.json() if assume_json else err.response.text()
```